tm2/pkg/crypto/secp256k1/secp256k1_nocgo_test.go

```diff
@@ -20,21 +19,25 @@ func TestSignatureVerificationAndRejectUpperS(t *testing.T) {
 		priv := GenPrivKey()
 		sigStr, err := priv.Sign(msg)
 		require.NoError(t, err)
-		sig := signatureFromBytes(sigStr)
-		require.False(t, sig.S.Cmp(secp256k1halfN) > 0)
+		_, ok := signatureFromBytes(sigStr)
+		require.True(t, ok)
```

Comment:
missing the previous comparison, nope?

Reply:
It is done in here: https://github.com/gnolang/gno/pull/1329/files#diff-15af116cb07c2b787dbd48a271f574b8de08b027ff307dc44534af8d4697c23eR59